{"@t":"2025-03-13T23:04:23.8688830Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":137,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-13T23:09:23.8924353Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":157,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-13T23:14:23.9312228Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":182,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-13T23:19:23.9583585Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":76,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-13T23:24:23.9761693Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":128,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-13T23:29:23.9894244Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-13T23:34:24.0174931Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":111,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-13T23:39:24.0367411Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":111,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-13T23:44:24.0647689Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":81,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-13T23:49:24.0736377Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":159,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-13T23:54:24.0867668Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":131,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-13T23:59:24.0920310Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":84,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T00:04:24.1189167Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":27,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T00:09:24.1248006Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":45,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T00:14:24.1336941Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":89,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T00:19:24.1364475Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":117,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T00:24:25.1631705Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":92,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T00:29:25.1739783Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":88,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T00:34:25.1862929Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":11,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T00:39:25.1929263Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":146,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T00:44:25.2032956Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":146,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T00:49:25.2266371Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":88,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T00:54:25.2445950Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":88,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T00:59:25.2717402Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":112,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T01:04:25.2926993Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":47,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T01:09:26.3131428Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":114,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T01:14:26.3351624Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":114,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T01:19:26.3420195Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":121,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T01:24:26.3756603Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":121,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T01:29:26.4071721Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":149,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T01:34:26.4133272Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":43,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T01:39:26.4228340Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":92,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T01:44:26.4395335Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":88,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T01:49:26.4451584Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":142,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T01:54:26.4604362Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":123,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T01:59:26.4764825Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":142,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T02:04:26.5206986Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":88,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T02:09:26.5511295Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":88,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T02:14:26.5631230Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":114,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T02:19:26.5853206Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":119,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T02:24:26.5956035Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":132,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T02:29:26.6237221Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":137,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T02:34:26.6401196Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":31,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T02:39:26.6563282Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":31,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T02:44:26.6681413Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":31,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T02:49:26.6829970Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":119,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T02:54:26.7038998Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":114,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T02:59:26.7238862Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":43,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T03:04:26.7554036Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":43,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T03:09:26.7718815Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":11,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T03:14:26.7903953Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":84,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T03:19:26.8052323Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":146,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T03:24:26.8165518Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":123,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T03:29:26.8265583Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":88,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T03:34:26.8319776Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":131,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T03:39:26.8456113Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":171,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T03:44:26.8598491Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":164,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T03:49:26.8619308Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":181,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T03:54:26.8686611Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":38,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T03:59:26.8921705Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T04:04:26.9259293Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":50,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T04:09:26.9293986Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":50,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T04:14:26.9432818Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":133,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T04:19:26.9501113Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":54,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T04:24:26.9624695Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":44,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T04:29:26.9720150Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":38,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T04:34:27.0029309Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":33,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T04:39:27.0158640Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":44,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T04:44:27.0282406Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":54,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T04:49:27.0504940Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":133,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T04:54:27.0674698Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":54,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T04:59:27.0751978Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T05:04:27.1021729Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":75,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T05:09:27.1168252Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":79,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T05:14:27.1723713Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":83,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T05:19:27.1841332Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":75,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T05:24:27.2019394Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":115,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T05:29:27.2600621Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":135,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T05:34:27.2632698Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":136,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T05:39:27.3007664Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":46,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T05:44:27.3898687Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":39,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T05:49:27.4047104Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":60,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T05:54:27.4077794Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":133,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T05:59:27.4187229Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":135,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T06:04:27.4417670Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":74,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T06:09:27.4542178Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":76,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T06:14:27.4784115Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://sitemaps.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":7384,"ProcessName":"w3wp","ThreadId":82,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2025-03-14T06:18:52.3881920Z","@mt":"New url {Url} detected, re-discovering application url.","Url":"http://MENTORPLAN.DK/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":7384,"ProcessName":"w3wp","ThreadId":101,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":5,"HttpRequestId":"54cc0113-aa63-475b-b41f-736bf2181210"}
{"@t":"2025-03-14T06:18:52.3881920Z","@mt":"ApplicationUrl: {UmbracoAppUrl} (UmbracoModule request)","UmbracoAppUrl":"http://MENTORPLAN.DK/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":7384,"ProcessName":"w3wp","ThreadId":101,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":5,"HttpRequestId":"54cc0113-aa63-475b-b41f-736bf2181210"}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Scheduled health check results:","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Folder & File Permissions","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Folder & File Permissions","HealthCheckResult":"Success","HealthCheckMessage":"All folders have the correct permissions set.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Folder & File Permissions","HealthCheckResult":"Success","HealthCheckMessage":"All files have the correct permissions set.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"SMTP Settings","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"WARN "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"SMTP Settings","HealthCheckResult":"Error","HealthCheckMessage":"In the Web.config file system.net/mailsettings section, the host is not configured.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Cross-site scripting Protection (X-XSS-Protection header)","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"WARN "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Cross-site scripting Protection (X-XSS-Protection header)","HealthCheckResult":"Error","HealthCheckMessage":"The header <strong>X-XSS-Protection</strong> was not found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Cookie hijacking and protocol downgrade attacks Protection (Strict-Transport-Security Header (HSTS))","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"WARN "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Cookie hijacking and protocol downgrade attacks Protection (Strict-Transport-Security Header (HSTS))","HealthCheckResult":"Error","HealthCheckMessage":"The header <strong>Strict-Transport-Security</strong> was not found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Content/MIME Sniffing Protection","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"WARN "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Content/MIME Sniffing Protection","HealthCheckResult":"Error","HealthCheckMessage":"The header or meta-tag <strong>X-Content-Type-Options</strong> used to protect against MIME sniffing vulnerabilities was not found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Excessive Headers","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Excessive Headers","HealthCheckResult":"Success","HealthCheckMessage":"No headers revealing information about the website technology were found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Click-Jacking Protection","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"WARN "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Click-Jacking Protection","HealthCheckResult":"Error","HealthCheckMessage":"The header or meta-tag <strong>X-Frame-Options</strong> used to control whether a site can be IFRAMEd by another was not found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"HTTPS Configuration","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"WARN "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Error","HealthCheckMessage":"You are currently not viewing the site using the HTTPS scheme.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Info","HealthCheckMessage":"The appSetting 'Umbraco.Core.UseHttps' is set to 'false' in your web.config file. Once you access this site using the HTTPS scheme, that should be set to 'true'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Success","HealthCheckMessage":"Your website's certificate is valid.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Database data integrity check","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database data integrity check","HealthCheckResult":"Success","HealthCheckMessage":"<p>All document paths are valid</p>\r\n","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database data integrity check","HealthCheckResult":"Success","HealthCheckMessage":"<p>All media paths are valid</p>\r\n","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Macro errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"WARN "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Macro errors","HealthCheckResult":"Error","HealthCheckMessage":"MacroErrors are set to 'throw' which will prevent some or all pages in your site from loading completely if there are any errors in macros. Rectifying this will set the value to 'inline'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Notification Email Settings","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"WARN "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Notification Email Settings","HealthCheckResult":"Error","HealthCheckMessage":"Notification email is still set to the default value of <strong>your@email.here</strong>.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Try Skip IIS Custom Errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Try Skip IIS Custom Errors","HealthCheckResult":"Success","HealthCheckMessage":"Try Skip IIS Custom Errors is set to 'true' and you're using IIS version '10.0'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Custom Errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Custom Errors","HealthCheckResult":"Success","HealthCheckMessage":"Custom errors are set to 'RemoteOnly'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Trace Mode","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Trace Mode","HealthCheckResult":"Success","HealthCheckMessage":"Trace mode is disabled.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Debug Compilation Mode","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T13:09:21.1794635Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Debug Compilation Mode","HealthCheckResult":"Success","HealthCheckMessage":"Debug compilation mode is disabled.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":7384,"ProcessName":"w3wp","ThreadId":19,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.5929755Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[ServerRegistration] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":121,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.5929755Z","@mt":"Stopping ({SignalSource})","SignalSource":"environment","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":7384,"ProcessName":"w3wp","ThreadId":121,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.5929755Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Examine shutting down","TimingId":"62ac02e","SourceContext":"Umbraco.Web.Search.ExamineComponent","ProcessId":7384,"ProcessName":"w3wp","ThreadId":121,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6085951Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ServerRegistration] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":117,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6085951Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[ServerRegistration] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":117,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6085951Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":14,"TimingId":"62ac02e","SourceContext":"Umbraco.Web.Search.ExamineComponent","ProcessId":7384,"ProcessName":"w3wp","ThreadId":121,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"Released from MainDom","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":7384,"ProcessName":"w3wp","ThreadId":121,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"Released ({SignalSource})","SignalSource":"environment","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":7384,"ProcessName":"w3wp","ThreadId":121,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[LogScrubber] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":121,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[ScheduledPublishing] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":121,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[RebuildIndexesOnStartup] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":121,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[KeepAlive] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":121,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[ScheduledTasks] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":121,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[TelemetryReporter] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":121,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[HealthCheckNotifier] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":121,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[ServerInstProcess] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":121,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[TempFileCleanup] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":121,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[Umbraco.Web.Scheduling.IBackgroundTask] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":121,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[LogScrubber] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":125,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[LogScrubber] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":125,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[RebuildIndexesOnStartup] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":127,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[RebuildIndexesOnStartup] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":127,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[KeepAlive] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":129,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[KeepAlive] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":129,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ScheduledTasks] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":129,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[ScheduledTasks] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":129,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ScheduledPublishing] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":49,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[ScheduledPublishing] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":49,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[Umbraco.Web.Scheduling.IBackgroundTask] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":49,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[Umbraco.Web.Scheduling.IBackgroundTask] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":49,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[TempFileCleanup] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[TempFileCleanup] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ServerInstProcess] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[ServerInstProcess] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[HealthCheckNotifier] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[HealthCheckNotifier] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[TelemetryReporter] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.6245625Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[TelemetryReporter] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":7384,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T18:06:19.7342475Z","@mt":"Application shutdown. Details: {ShutdownReason}\r\n\r\n_shutDownMessage={ShutdownMessage}\r\n\r\n_shutDownStack={ShutdownStack}","ShutdownReason":"HostingEnvironment","ShutdownMessage":"HostingEnvironment initiated shutdown\r\nHostingEnvironment caused shutdown","ShutdownStack":"   at System.Environment.GetStackTrace(Exception e, Boolean needFileInfo)\r\n   at System.Environment.get_StackTrace()\r\n   at System.Web.Hosting.HostingEnvironment.InitiateShutdownInternal()\r\n   at System.Web.Hosting.HostingEnvironment.InitiateShutdownWithoutDemand()\r\n   at System.Web.Hosting.PipelineRuntime.StopProcessing()","SourceContext":"Umbraco.Web.UmbracoApplicationBase","ProcessId":7384,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:14:14.0496377Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Booting Umbraco 8.17.0.","TimingId":"201b886","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":29592,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"f44912d4-8f5c-4377-b950-57fca105803c"}
{"@t":"2025-03-14T19:14:14.0652747Z","@mt":"Booting site '{HostingSiteName}', app '{HostingApplicationID}', path '{HostingPhysicalPath}', server '{MachineName}'.","HostingSiteName":"mentorplan.dk","HostingApplicationID":"/LM/W3SVC/844263/ROOT","HostingPhysicalPath":"d:\\web\\localuser\\mentorplan.dk\\public_html\\","MachineName":"NT45","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":29592,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"f44912d4-8f5c-4377-b950-57fca105803c"}
{"@t":"2025-03-14T19:14:14.0825012Z","@mt":"Acquiring.","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":29592,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"f44912d4-8f5c-4377-b950-57fca105803c"}
{"@t":"2025-03-14T19:14:14.0825012Z","@mt":"Acquired.","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":29592,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"f44912d4-8f5c-4377-b950-57fca105803c"}
{"@t":"2025-03-14T19:14:14.4764537Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Resolving composer types.","TimingId":"713323f","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":29592,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"f44912d4-8f5c-4377-b950-57fca105803c"}
{"@t":"2025-03-14T19:14:14.4920932Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Resolved.","Duration":10,"TimingId":"713323f","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":29592,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"f44912d4-8f5c-4377-b950-57fca105803c"}
{"@t":"2025-03-14T19:14:14.8236191Z","@mt":"Profiler is VoidProfiler, not profiling (must run debug mode to profile).","SourceContext":"Umbraco.Web.Logging.WebProfilerComponent","ProcessId":29592,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"f44912d4-8f5c-4377-b950-57fca105803c"}
{"@t":"2025-03-14T19:14:14.8712552Z","@mt":"Initializing Umbraco internal event handlers for cache refreshing.","SourceContext":"Umbraco.Web.Cache.DistributedCacheBinderComponent","ProcessId":29592,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"f44912d4-8f5c-4377-b950-57fca105803c"}
{"@t":"2025-03-14T19:14:14.8861131Z","@mt":"Adding examine event handlers for {RegisteredIndexers} index providers.","RegisteredIndexers":3,"SourceContext":"Umbraco.Web.Search.ExamineComponent","ProcessId":29592,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"f44912d4-8f5c-4377-b950-57fca105803c"}
{"@t":"2025-03-14T19:14:14.9017501Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Booted.","Duration":849,"TimingId":"201b886","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":29592,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"f44912d4-8f5c-4377-b950-57fca105803c"}
{"@t":"2025-03-14T19:14:16.2956841Z","@mt":"New url {Url} detected, re-discovering application url.","Url":"http://mentorplan.dk/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":29592,"ProcessName":"w3wp","ThreadId":13,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"77866d10-1268-49cb-a025-530bbe21a2b3"}
{"@t":"2025-03-14T19:14:16.2956841Z","@mt":"ApplicationUrl: {UmbracoAppUrl} (UmbracoModule request)","UmbracoAppUrl":"http://mentorplan.dk/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":29592,"ProcessName":"w3wp","ThreadId":13,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"77866d10-1268-49cb-a025-530bbe21a2b3"}
{"@t":"2025-03-14T19:14:17.2294058Z","@mt":"Registered with MainDom, localContentDbExists? {LocalContentDbExists}, localMediaDbExists? {LocalMediaDbExists}","LocalContentDbExists":true,"LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":29592,"ProcessName":"w3wp","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"6c105b24-894b-467c-a6df-68b7ff2ba341"}
{"@t":"2025-03-14T19:14:17.2294058Z","@mt":"Creating the content store, localContentDbExists? {LocalContentDbExists}","LocalContentDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":29592,"ProcessName":"w3wp","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"6c105b24-894b-467c-a6df-68b7ff2ba341"}
{"@t":"2025-03-14T19:14:17.2294058Z","@mt":"Creating the media store, localMediaDbExists? {LocalMediaDbExists}","LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":29592,"ProcessName":"w3wp","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"6c105b24-894b-467c-a6df-68b7ff2ba341"}
{"@t":"2025-03-14T19:14:17.4092032Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading content from local cache file","TimingId":"eb55a3d","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":29592,"ProcessName":"w3wp","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"6c105b24-894b-467c-a6df-68b7ff2ba341"}
{"@t":"2025-03-14T19:14:17.4248284Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":8,"TimingId":"eb55a3d","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":29592,"ProcessName":"w3wp","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"6c105b24-894b-467c-a6df-68b7ff2ba341"}
{"@t":"2025-03-14T19:14:17.4248284Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading media from local cache file","TimingId":"b22d64a","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":29592,"ProcessName":"w3wp","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"6c105b24-894b-467c-a6df-68b7ff2ba341"}
{"@t":"2025-03-14T19:14:17.4248284Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":0,"TimingId":"b22d64a","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":29592,"ProcessName":"w3wp","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"6c105b24-894b-467c-a6df-68b7ff2ba341"}
{"@t":"2025-03-14T19:14:17.4560773Z","@mt":"Starting initialize async background thread.","SourceContext":"Umbraco.Web.Search.BackgroundIndexRebuilder","ProcessId":29592,"ProcessName":"w3wp","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"6c105b24-894b-467c-a6df-68b7ff2ba341"}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Scheduled health check results:","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Folder & File Permissions","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Folder & File Permissions","HealthCheckResult":"Success","HealthCheckMessage":"All folders have the correct permissions set.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Folder & File Permissions","HealthCheckResult":"Success","HealthCheckMessage":"All files have the correct permissions set.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"SMTP Settings","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"WARN "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"SMTP Settings","HealthCheckResult":"Error","HealthCheckMessage":"In the Web.config file system.net/mailsettings section, the host is not configured.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Cross-site scripting Protection (X-XSS-Protection header)","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"WARN "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Cross-site scripting Protection (X-XSS-Protection header)","HealthCheckResult":"Error","HealthCheckMessage":"The header <strong>X-XSS-Protection</strong> was not found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Cookie hijacking and protocol downgrade attacks Protection (Strict-Transport-Security Header (HSTS))","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"WARN "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Cookie hijacking and protocol downgrade attacks Protection (Strict-Transport-Security Header (HSTS))","HealthCheckResult":"Error","HealthCheckMessage":"The header <strong>Strict-Transport-Security</strong> was not found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Content/MIME Sniffing Protection","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"WARN "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Content/MIME Sniffing Protection","HealthCheckResult":"Error","HealthCheckMessage":"The header or meta-tag <strong>X-Content-Type-Options</strong> used to protect against MIME sniffing vulnerabilities was not found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Excessive Headers","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Excessive Headers","HealthCheckResult":"Success","HealthCheckMessage":"No headers revealing information about the website technology were found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Click-Jacking Protection","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"WARN "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Click-Jacking Protection","HealthCheckResult":"Error","HealthCheckMessage":"The header or meta-tag <strong>X-Frame-Options</strong> used to control whether a site can be IFRAMEd by another was not found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"HTTPS Configuration","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"WARN "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Error","HealthCheckMessage":"You are currently not viewing the site using the HTTPS scheme.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Info","HealthCheckMessage":"The appSetting 'Umbraco.Core.UseHttps' is set to 'false' in your web.config file. Once you access this site using the HTTPS scheme, that should be set to 'true'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Success","HealthCheckMessage":"Your website's certificate is valid.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Database data integrity check","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database data integrity check","HealthCheckResult":"Success","HealthCheckMessage":"<p>All document paths are valid</p>\r\n","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database data integrity check","HealthCheckResult":"Success","HealthCheckMessage":"<p>All media paths are valid</p>\r\n","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Macro errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"WARN "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Macro errors","HealthCheckResult":"Error","HealthCheckMessage":"MacroErrors are set to 'throw' which will prevent some or all pages in your site from loading completely if there are any errors in macros. Rectifying this will set the value to 'inline'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Notification Email Settings","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"WARN "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Notification Email Settings","HealthCheckResult":"Error","HealthCheckMessage":"Notification email is still set to the default value of <strong>your@email.here</strong>.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Try Skip IIS Custom Errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Try Skip IIS Custom Errors","HealthCheckResult":"Success","HealthCheckMessage":"Try Skip IIS Custom Errors is set to 'true' and you're using IIS version '10.0'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Custom Errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Custom Errors","HealthCheckResult":"Success","HealthCheckMessage":"Custom errors are set to 'RemoteOnly'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Trace Mode","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Trace Mode","HealthCheckResult":"Success","HealthCheckMessage":"Trace mode is disabled.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Debug Compilation Mode","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2025-03-14T19:17:17.6722535Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Debug Compilation Mode","HealthCheckResult":"Success","HealthCheckMessage":"Debug compilation mode is disabled.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29592,"ProcessName":"w3wp","ThreadId":23,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
